tsconfig.json

```diff
@@ -1,17 +1,19 @@
 {
+	"$schema": "https://json.schemastore.org/tsconfig",
 	"compilerOptions": {
-		"module": "commonjs",
+		"skipLibCheck": true,
```

Comment:
Not added, simply moved up as it will be simpler for me to find which rule is still missing if I preserve the order from my reference.